attributes/source.rb

```diff
@@ -17,7 +17,13 @@
 # limitations under the License.
 #
 
-default['pdns']['source']['version'] = '3.4.7'
-default['pdns']['source']['url'] = "https://downloads.powerdns.com/releases/pdns-#{node['pdns']['source']['version']}.tar.bz2"
 default['pdns']['source']['path'] = '/opt'
-default['pdns']['source']['backends'] = %w( bind )
+
+case node['pdns']['flavor']
```

Comment:
I would change this from a case to an if/else and as mentioned earlier, want to avoid use of the node object in attributes as it prevents wrapper cookbooks since it gets evaluated too soon.